<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="12,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="CTImageViewer" name="drawingArea">
          <property name="minimumSize">
           <size>
            <width>512</width>
            <height>512</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="leftLoadImageButtonSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="loadImageButton">
          <property name="text">
           <string>load Image</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="rightLoadImageButtonSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="windowingCheckbox">
          <property name="text">
           <string>Windowing</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
          <item>
           <widget class="QLabel" name="windowCenterLabel">
            <property name="text">
             <string>Windowing Center</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="windowingCenterValue">
            <property name="text">
             <string>0</string>
            </property>
            <property name="maxLength">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="windowCenterSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-1024</number>
          </property>
          <property name="maximum">
           <number>3095</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
          <item>
           <widget class="QLabel" name="windowWidthLabel">
            <property name="text">
             <string>Windowing Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="windowingWidthValue">
            <property name="text">
             <string>800</string>
            </property>
            <property name="maxLength">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="windowWidthSlider">
          <property name="maximum">
           <number>2059</number>
          </property>
          <property name="value">
           <number>800</number>
          </property>
          <property name="sliderPosition">
           <number>800</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TickPosition::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="windowingSliderSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTImageViewer</class>
   <extends>QLabel</extends>
   <header>ctimageviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
